src/proteingym/base/measurement.py

```diff
@@ -0,0 +1,194 @@
+"""The measurements on which the assays are based."""
+
+import dataclasses
+from enum import StrEnum
+from pathlib import Path
+
+import polars as pl
+from pydantic import (
+    BaseModel,
+    ConfigDict,
+    FilePath,
+    SerializationInfo,
+    ValidationInfo,
+    field_serializer,
+    field_validator,
+    model_validator,
+)
+
+
+class MeasurementsFormat(StrEnum):
+    """Supported assay file formats."""
+
+    CSV = ".csv"
+    """A comma separated text file"""
+
+
+@dataclasses.dataclass(kw_only=True, frozen=True)
+class Field:
```

Comment:
I find `Field` unfortunate in a package that uses pydantic. I proposed `AssayField` in #374 - can we merge that first? I know it may read double in the toml `assay.assay_field` instead of `assay.field` but stil better pydantic clash imo. 

- Column: no - refers to a 2D layout, says nothing.
- Variable: already use that
- Observable: bit cryptic
- Target: not necessarily a prediction target anymore
- AssayField: bit double since used with Assay
- DataField: ?
- ..

@karel-w